results/views.py

```diff
@@ -498,6 +498,8 @@ def mark_pages(canvas_mark, direction: Napravleniya):
         canvas_mark.line(55 * mm, 11.5 * mm, 181 * mm, 11.5 * mm)
         canvas_mark.restoreState()
 
+    count_pages = 0
+    is_PageBreak = False
```

Comment:
Это не класс и не очень понятно, что is) поэтому предложение такое
```suggestion
    has_page_break = False
```